src/status_im2/contexts/chat/composer/reply/style.cljs

```diff
@@ -6,8 +6,9 @@
   [pin? in-chat-input?]
   {:flex-direction :row
    :height         (when-not pin? 24)
-   :margin-left    (if (and (not in-chat-input?) (not pin?)) 26 (if platform/android? 4 0))
+   :margin-left    (when (and (not in-chat-input?) (not pin?)) 26)
```

Comment:
Are you sure about this change?

Reply:
Tested the flow thoroughly after changing it and was able to not reproduce the issue and the switcher wasn't visible inside composer anymore.

Reply:
It matches the design better without it.